src/Microsoft.Azure.SignalR.Common/ServiceConnections/MultiEndpointServiceConnectionContainer.cs

```diff
@@ -305,7 +305,7 @@ private async Task RemoveHubServiceEndpointAsync(HubServiceEndpoint endpoint)
                     return;
                 }
 
-                _ = container.ConnectionContainer.OfflineAsync(false);
+                _ = container.ConnectionContainer.OfflineAsync(GracefulShutdownMode.Off);

```

Comment:
GracefulShutdownMode.Off would allow clients to re-connect as we're shutting down the service connections?

Reply:
Yes, it will act the same as the current behavior, while `On` represents keeping client connections alive for a certain period of time, and `Migrate` represents migrating client connections to another valid server.